src/figma/tokens.json

```
@@ -828,29 +828,39 @@
           "type": "color"
         },
         "default-hover": {
-          "value": "#FAFBFC",
-          "description": "(grey030: #FAFBFC) (grey030: #FAFBFC) For component hover states that use background/default",
-          "type": "color"
+          "value": "#FAFAFA",
+          "type": "color",
+          "description": "(#FAFAFA)For component hover states that use background/default"
         },
         "default-pressed": {
-          "value": "#FAFBFC",
-          "description": "(grey030: #FAFBFC) For component pressed states that use background/default",
-          "type": "color"
+          "value": "#EBEBEB",
+          "type": "color",
+          "description": "(#EBEBEB) For component pressed states that use background/default"
         },
         "alternative": {
           "value": "#F2F4F6",
           "description": "(grey040: #F2F4F6) For a subtle contrast option for neutral backgrounds. (Example: backdrop, header background)",
           "type": "color"
         },
         "alternative-hover": {
-          "value": "#D6D9DC",
-          "description": "(grey100: #D6D9DC) For component hover states that use background/alternative",
-          "type": "color"
+          "value": "#EDEFF1",
+          "type": "color",
+          "description": "(#EDEFF1) For component hover states that use background/alternative"
         },
         "alternative-pressed": {
-          "value": "#D6D9DC",
-          "description": "(grey100: #D6D9DC) For component pressed states that use background/alternative",
-          "type": "color"
+          "value": "#DFE0E2",
+          "type": "color",
+          "description": "(#DFE0E2) For component pressed states that use background/alternative"
+        },
+        "hover": {
+          "value": "#00000005",
+          "type": "color",
+          "description": "For component hover states with no background color"
```

Add hex value so it shows up on hover of the token in Figma

```suggestion
          "description": "(#00000005) For component hover states with no background color"
```